targets/TARGET_Cypress/TARGET_PSOC6/TARGET_CY8CKIT_062_WIFI_BT_PSA/prebuilt/README.md

```diff
@@ -7,7 +7,7 @@ These images were compiled by the following command:
 
 ```
 mbed compile -m CY8CKIT_062_WIFI_BT_M0_PSA -t GCC_ARM --profile release -N psa_release_1.0
-mbed test --compile -m CY8CKIT_062_WIFI_BT_M0_PSA -t GCC_ARM --profile debug -n *psa-spm*,*psa-crypto_access_control
+mbed test --compile -m CY8CKIT_062_WIFI_BT_M0_PSA -t GCC_ARM --profile release -n *psa-spm*,*psa-crypto_access_control
```

Comment:
Since https://github.com/ARMmbed/mbed-os/commit/04c58031310c6eb214600eb1cac4368e3454c091, debug binaries no longer fit into CM0+ address space, hence switched to release profile.

Reply:
please use `python tools/psa/release.py -m CY8CKIT_062_WIFI_BT_M0_PSA ` instead of the 2 coomands

Reply:
please replace both commands with `python tools/psa/release.py -m CY8CKIT_062_WIFI_BT_M0_PSA `

Reply:
Fixed in the https://github.com/ARMmbed/mbed-os/pull/10308/commits/83275da0cf4d25a4b8da3d9ed7f195188e76de69 commit